app/src/main/java/com/onboarding/parkingsystemkotlin/utils/CalendarUtils.kt

```diff
@@ -0,0 +1,21 @@
+package com.onboarding.parkingsystemkotlin.utils
+
+import androidx.constraintlayout.solver.widgets.ConstraintAnchor
+import java.util.Calendar
+
+object CalendarUtils {
+    fun getDateString(date: Calendar?): String {
+        var ret = ConstantUtils.EMPTY_STRING
+        date?.let {
+            val day = it.get(Calendar.DAY_OF_MONTH)
+            val month = (it.get(Calendar.MONTH) + ConstantUtils.MONTH_ADJUSTMENT)
```

Comment:
pls use something like this 
`val month = ${it.get(Calendar.MONTH)}${ConstantUtils.MONTH_ADJUSTMENT}`

Reply:
that + is not a concatenation, its actually a sum